### DISCUSSION TITLE
como colocar os arquivo no cartão Carteador

### DISCUSSION THREAD

**Discussion by rodrigoff40** [sanni/cartreader#76] [Need help]
como faço para colocar os arquivo na Raiz do pendriver não estou consseguindo.

**Comment by sanni** [comment]
![ACCVAVDGDFHSBS](https://user-images.githubusercontent.com/19169494/104836491-90b0ed00-58ae-11eb-83a4-9d0a14c8005b.jpg)

**Comment by rodrigoff40** [comment]
ola Sanni consegui ja coloquei os arquivo, as roms é colocada na Rais do Pendriver também, e elas pode ser baixada da net, e pode esta zipada ou não.

**Comment by sanni** [comment]
The Cart Reader can't read zipped files. Extract them first.
